### PR TITLE
Add support for PHP 8.1

### PR DESCRIPTION
**Pull request by eduardomozart**
This PR also includes the fixes available at #279.

### CODE REVIEW

File: ODT/ODTUtility.php [hunk 1]
```diff
@@ -332,14 +332,14 @@ public static function adjustValuesForODT (&$properties, ODTUnits $units, $maxWi
...
-            $properties ['text-underline-style'] = 'solid';
-        }
-        if ( $properties ['text-decoration'] == 'overline' ) {
-            $properties ['text-overline-style'] = 'solid';
+        if (isset($properties ['text-decoration'])) {
+            switch ($properties ['text-decoration']) {
+                case 'line-through':
+                case 'underline':
+                case 'overline':
+                    $properties ['text-line-through-style'] = 'solid';
```
Comment (Klap-in) [inline ODT/ODTUtility.php:340]: This is now for all cases assigned to the same entry, please note that ‘…line-through…’ part varies.

**Review by Klap-in — COMMENTED** [review]
Looks good, thanks!
I observed one issue.

I haven’t tested this. Did you test this yourself?

File: ODT/ODTUtility.php (hunk 1 above)
Comment (eduardomozart) [reply ODT/ODTUtility.php]: Sorry, while fixing the bugs I read only the 'solid' value and didn't check that the property was different.

**Comment by Klap-in** [comment]
Thanks!

**Pull request merged**